packages/terra-site/src/examples/action-footer/ActionFooterComponentNavigation.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import List from 'terra-list';
+import styles from '../../site.scss';
+
+const propTypes = {
+  onClick: PropTypes.func,
+};
+
+const ActionFooterComponentNavigation = ({
+    onClick,
+  }) =>
```

Comment:
nit: can we please wrap this in parenthesis?

Reply:
Interesting.. Not sure how that slipped through 🤷‍♂️

Reply:
hash-tag:who knows